test/deployment/resources/setup.go

```diff
@@ -0,0 +1,26 @@
+package resources
+
+import (
+	"github.com/artemiscloud/activemq-artemis-operator-test-suite/test"
+	"github.com/onsi/ginkgo"
+)
+
+// Constants available for all test specs related with the One Interior topology.
+
+const (
+	DeployName = "resources"
+	BaseName   = "broker-framework"
+)
+
+var (
+	sw *test.SetupWrapper
+)
+
+var _ = ginkgo.BeforeEach(func() {
+	sw = &test.SetupWrapper{}
+	sw.WithBaseName(BaseName).WithDeployName(DeployName)
+	if !test.Config.NeedsLatestCR {
+		ginkgo.Skip("Not supported on pre-0.17 operator")
```

Comment:
Is this check sufficient to verify operator version >= 17?

Reply:
This check just verify if the v2 flag was used, so, I think it allow the execution if for some reason the v2alpha2 CRD is provided with the v2 flag and it fail. 
One possible solution that I see is to check the loaded CRD version instead of just assume the correct one was specified.